pages/users/index.md

```diff
@@ -1 +1,17 @@
-# Welcome
\ No newline at end of file
+# Welcome
+
```

Comment:
Add a welcome message like "Welcome to the XRPL EVM documentation here you can find..."